apps/docs/app/ui/components/migrateGuide/migrate-guide.css

```diff
@@ -0,0 +1,9 @@
+.hd-migrate-guide code {
```

Comment:
seems to be a lot of stuff from your other pr

Reply:
Yep, my bad :( I think the best course of action is to merge the two other PR's then merge main in this one that way only the relevant changes will show.